doc/docker-compose/Makefile(sample)

```diff
@@ -0,0 +1,709 @@
+# /*******************************************************************************
```

Comment:
Add TODO to remove these `docker-compose` files once service accepted into EdgeX Foundry and added to compose builder.

Reply:
Added the comment.